charts/kubearchive/values.yaml

```diff
@@ -93,3 +88,13 @@ database:
     dbUser: "ps_user"
     # database password
     dbPassword: "P0stgr3sdbP@ssword"  # notsecret
+
+observability:
```

Comment:
I was just thinking about this myself.  I think we should create an `integrations` key and put `observability` under that.  That would also allow it to be used to contain `splunk` or any other integration we support.

Reply:
I think it makes sense, we can always go back if we find something wrong with the nesting.

Reply:
Introduced an `integrations` key.